cves/CVE-2010-1157.yml

```diff
@@ -220,7 +208,7 @@ mistakes:
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
   answer: |
-    It seems that this vulnerability was an oversight with regards to how the configuration and authentication code
-    interacted. The fix was to provide a default realm name that did not contain sensitive data, instead of 
-    generating a string that used the actual hostname and port. What lead to this vulnerability was most likely
-    a simple ignorance to the value of a server's hostname information.
+    This vulnerability seemed to be a result of a design mistake. The developers
+    should have seen that giving out the IP address of the Tomcat machine so easily
+    was a risk. Overall the team decided to scrap the output and switched
+    it over to something that was much more safe (a default hard-coded string).
```

Comment:
Switched it over to something safer vs something that was much more safe. Clearer and easier to read.

Reply:
Updated